src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ImplementIDisposableCorrectly.cs

```diff
@@ -14,6 +17,9 @@ public abstract class ImplementIDisposableCorrectlyAnalyzer : DiagnosticAnalyzer
     {
         internal const string RuleId = "CA1063";
 
+        private const string HelpLinkUri = "https://msdn.microsoft.com/library/ms244737.aspx";
```

Comment:
You can make this type abstract and delete the C# and VB derived class stubs that we have.